scripts/benchmark_stats.py

```diff
@@ -69,6 +76,12 @@ def _process_file_data(self):
                         )
                         self._total_failures += int(failure_count)
                     else:
+                        if row["Name"] == "/submitted/download-pdf":
+                            self._pdf_percentile.append(int(row.get("99%")))
+
+                        if row["Name"] == "/session":
+                            self._session_percentile.append(int(row.get("99%")))
```

Comment:
Same as [the other comment](https://github.com/ONSdigital/eq-survey-runner-benchmark/pull/164/files#r1714766385).

Reply:
Updated 👍